CHANGELOG.md

```diff
@@ -71,3 +71,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Safari crashes when remote video tiles are added or toggled
 
 ## [1.0.0] - 2019-11-20
+ 
```

Comment:
Perhaps we could write a more meaningful changelog such as "Fix canary deployment script"?

Reply:
Yes will update it